HaroldAdviser/Views/User/AllRepositories.cshtml

```diff
@@ -11,4 +11,5 @@
 @section scripts
     {
     <script src="~/Scripts/repository-check.js"></script>
+    @*<script src="~/Scripts/get-warnings.js"></script>*@
```

Comment:
Please remove comment. Also, this script is for repository view page. Please move it here.